www/templates/intro/summary.html

```diff
@@ -16,6 +16,8 @@
 	<div class="intro-space"></div>
 	<li> Provide <b>aggregate data on travel patterns</b> so that cities can improve their
 	land use and transportation planning.
+	<div class="intro-space"></div>
```

Comment:
Don't you want to remove the existing paragraphs? You don't currently display transportation carbon footprint, suggestions on how to change travel patterns or aggregate data...